procurement_purchase_no_grouping/models/purchase_order.py

```diff
@@ -17,6 +17,17 @@ def search(self, cr, uid, args, offset=0, limit=None, order=None,
         if (context and context.get('grouping', 'standard') == 'order' and
                 make_po_conditions.issubset(set(x[0] for x in args))):
             return []
+        # If grouping is one purchase order from same sale order
+        if (context and
+                context.get('grouping', 'standard') == 'one_sale_one_purchase'
+                and make_po_conditions.issubset(set(x[0] for x in args))):
+            '''
+                Add new condition to domain, becouse I need find only one
+                purchase order with de same suplier and origin to add new line
+            '''
+            aux = args
+            aux.append(('origin', '=', context.get('origin', False)))
+            arg = aux
```

Comment:
This is an Error because arsg is a list an in Python if I assign list = list is by reference, for that this works ok. But I clean the code now.